orderly_web/start.py

```diff
@@ -57,6 +56,16 @@ def orderly_container(cfg, docker_client):
     return container
 
 
+def orderly_initial_data(cfg, container):
+    if not orderly_is_initialised(container):
```

Comment:
So if you've already initialised the volume, it ignores the initial configuration. I guess if you decide you want to re-start from a different repo or whatever you could just make a new config path. Maybe a log message that initial config (if present) is being ignored because already initialised would help to make this really clear to user?

Reply:
More logging added in f6fdb5a